### DISCUSSION TITLE
Version 3.2.0 is now available.

### DISCUSSION THREAD

**Discussion by janiversen** [pymodbus-dev/pymodbus#1418] [Announcements]
Please refer to CHANGELOG and release notes for changes.

API_changes.rst contains changes that potentially affect applications using pymodbus.

**Comment by janiversen** [comment]
The next works in progress are mostly simplifying and unifying the client code, possibly by having a shared transport layer with the server.

And of course we will continue to expand the use of mypy and other control tools...as well as fixing bugs.

Target for v3.3.0 is no earlier than June.

**Comment by jamesbraza** [comment]
Question: what was the purpose of the `logging.StreamHandler()` added [here](https://github.com/pymodbus-dev/pymodbus/pull/1411/files#diff-2e34510eb44ba353d136b13b619671e1e1fe8d401206187be182021bf51df986R46)? (@janiversen)

**Reply by janiversen** [reply]
to solve a problem.
